docs/book/src/reference/versions.md

```diff
@@ -82,6 +82,7 @@ These diagrams show the relationships between components in a Cluster API releas
 | Kubernetes v1.26  | ✓                    | ✓                 | ✓                 | ✓                 |
 | Kubernetes v1.27  | ✓                    | ✓                 | ✓                 | ✓                 |
 | Kubernetes v1.28  |                      | ✓                 | ✓                 | ✓                 |
+| Kubernetes v1.29  |                      |                   |                   | ✓                 |
```

Comment:
I guess let's also add it to 1.6 (if it's not part of a different PR). We should also add 1.29 to the 1.6 column on the release-1.6 branch